Source/Curves/ContinuousCurves/EquationEvaluatorClass.f90

```diff
@@ -1018,17 +1019,17 @@ END SUBROUTINE NumberStackPop
 !
 !////////////////////////////////////////////////////////////////////////
 !
-      SUBROUTINE NumberStackPeek( stack, v ) 
-         IMPLICIT NONE 
-         TYPE(NumberStack) :: stack
-         REAL(KIND=EP)     :: v
-         
-         IF( stack % top <= 0 ) THEN
-            PRINT *, "Attempt to peek from empty number stack"
-         ELSE 
-            v = stack % values(stack % top)
-         END IF
-      END SUBROUTINE NumberStackPeek
+!      SUBROUTINE NumberStackPeek( stack, v ) 
+!         IMPLICIT NONE 
+!         TYPE(NumberStack) :: stack
+!         REAL(KIND=EP)     :: v
+!         
+!         IF( stack % top <= 0 ) THEN
+!            PRINT *, "Attempt to peek from empty number stack"
+!         ELSE 
+!            v = stack % values(stack % top)
+!         END IF
+!      END SUBROUTINE NumberStackPeek
 !
 !////////////////////////////////////////////////////////////////////////
 !
```

Comment:
I believe this routine is for debugging. It is also present in `main` but unused.

Reply:
This routine is a standard stack procedure, which just happens not to be used (so far) in this class. This code predates FTObjectLibrary by many years and implements its own stack. Also makes it standalone. My rule has generally been to never to delete code, because someday, invariably, I wish I hadn't. Should someone in the future need to peek, they would never know that it had been once in the file and deleted, unless there is a comment placed here noting the fact that it was, but deleted in a particular commit. But then the source might as well be left in. I removed it because it isn't used or covered by the tests.

Reply:
OK, fair enough.